refactor(render): move services to right location

tbosch opened the pull request:
core/compiler/events -> render/dom/events
core/compiler/url_resolver -> services/url_resolver
core/compiler/xhr/\* -> services/*

yjbanov commented:
LGTM except that files do not register as moved, but removed and added, therefore losing history. I suggest splitting into 2 commits: move, fix imports.

tbosch committed:
refactor(render): remove duplicate files to prepare for move

Remove first so Github shows the files as being moved instead of copied and deleted.

tbosch committed:
refactor(render): move services to right location
    
core/compiler/events -> render/dom/events
core/compiler/url_resolver -> services/url_resolver
core/compiler/xhr/* -> services/*
